roxie/topo/toposerver.cpp

```diff
@@ -269,6 +269,16 @@ void doServer(ISocket *socket)
     pinger.join();
 }
 
+static constexpr const char * defaultJson = R"!!({
+  "version": "1.0",
+  "TopoServer": {
+    "logdir": "",
+    "port": 9004,
+    "stdlog": true,
+    "traceLevel": 1,
```

Comment:
Is the trailing comma a potential issue?

Reply:
No but I fixed anyway